integration/simulation/validate_chain.go

```diff
@@ -273,7 +275,7 @@ func checkBlockchainOfObscuroNode(
 	totalAmountInSystem := s.Stats.TotalDepositedAmount - totalSuccessfullyWithdrawn
 	total := uint64(0)
 	for _, wallet := range s.Params.Wallets.SimObsWallets {
-		total += balance(nodeClient, wallet.Address())
+		total += balance(nodeClient, wallet.Address(), s.Params.Wallets.Tokens[evm.BTC].L2ContractAddress)
```

Comment:
Don't we need some validation around the `evm.ETH` tokens? Or do we at least need a TODO to add it?

Reply:
part of the next work.